src/data/components/marketing-stats.mdx

```diff
@@ -10,6 +10,12 @@ components:
   1:
     title: Simple with Grid
     dark: true
+  2: 
+    title: Simple with Flex
+    dark: true
+  3:
+    title: Simple With Flex
+    dark: true
```

Comment:
- Duplicate `title`
- Non-descriptive

If it makes it easier, feel free to rename the first component `title`.